news/2019-07-07-beatmap-spotlights-spring-2019.md

```diff
@@ -0,0 +1,306 @@
+
+_written by [Kazu](https://osu.ppy.sh/users/920861)
+
+[![](/wiki/shared/news/2019-07-07-beatmap-spotlights-spring-2019/taiko/sugary.jpg)](https://osu.ppy.sh/beatmapsets/578489)
+
+If you are looking for stamina challenges for this season, Zekk's "Sugary Daydream", hosted by [TKS](https://osu.ppy.sh/users/940878) with the help of [Raiden](https://osu.ppy.sh/users/Raiden) and [\[Chichinya\]](https://osu.ppy.sh/users/2140739), is likely a promising choice with the high BPM nature.
+
+Being a very experienced mapper in the Taiko community since their appearance in 2013, TKS never ceases their mapping enthusiasm to amaze people with this thrilling creation. The top difficulty features 1/6 and 1/8 bursts that arrange incredibly well with the packed 1/4 patterning, while comfortable density transitions are shown correspondingly among sessions with a different pace. High consistency is also maintained throughout the set with a relatively reasonable spread spectrum, making it fun and satisfying to play.
+
+The daydream is apparently sugary, but I recommend you to brace yourself for the real bitter nightmare it envelops.
+
+_written by [Nardoxyribonucleic](https://osu.ppy.sh/users/876419)
+
+[![](/wiki/shared/news/2019-07-07-beatmap-spotlights-spring-2019/taiko/city.jpg)](https://osu.ppy.sh/beatmapsets/588177)
+
+Since the "City Never Sleeps", you know something's keeping it up on its toes, and this rendition of Dirty Android's track mapped by [Lyawi](https://osu.ppy.sh/users/5851253) does just about that as well!
+
+The thing that's keeping you on your toes while playing is not an insanely difficult map in this case, however. What sets this set apart from others is that each difficulty provides a good challenge for all beginners trying to advance to the next level, as each of them is mapped in a fairly dense manner while maintaining good flow and going along with the song nicely. [Stefan](https://osu.ppy.sh/users/626907)'s Guest Difficulty further assists the idea of that set as well.
+
+Generally, the set as a whole is as clean as it can get and it's a great map to debut within the Ranked section and the Spotlights as a mapper!
+
+_written by [Greenshell](https://osu.ppy.sh/users/8693851)
+
+[![](/wiki/shared/news/2019-07-07-beatmap-spotlights-spring-2019/taiko/flags.jpg)](https://osu.ppy.sh/beatmapsets/950459)
+
+One of [Axer](https://osu.ppy.sh/users/7299864)'s most eccentric maps, "Flags on the Moon" does a really good job at staying true to the song through all these rough time signatures, syncopations and multiple changes of rhythm that make it very difficult to follow through; which ends up being a challenge to time perfectly for these steady beat accuracy players. 
+
+For this reason, Axer has his place on this season spotlights very well deserved, for his advanced patterning skills and carefully polished gameplay-centred mapping style.
```

Comment:
```suggestion
For this reason, Axer has a very well deserved his place on this season's spotlights for his advanced patterning skills and carefully polished gameplay-centred mapping style.
```

Reply:
“Axer has a very well deserved his place”
I think instead should be
“Axer has a very well deserved place”
On mobile rn sorry for not writing this as a change suggestion